pkgs/misc/drivers/xow/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, libusb, gitMinimal }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "medusalix";
+    repo = "xow";
+    rev = "v${version}";
+    sha256 = "03ajal91xi52svzy621aa4jcdf0vj4pqd52kljam0wryrlmcpbr3";
+  };
+
+  makeFlags = "BUILD=RELEASE";
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ gitMinimal ];
```

Comment:
The code obtained by `fetchFromGitHub` does not contain `.git` directory by default so https://github.com/medusalix/xow/blob/897eee07351f88a04eb16a02b8bca0b91f91cad4/Makefile#L4 will  be useless. Setting `"VERSION=${version}"` in `makeFlags` might be more viable strategy.

Reply:
Good point. If I set `VERSION` as a `makeFlag`, the build doesn't require `gitMinimal` anymore.